pathfinding/pathfinder.py

```diff
@@ -1,29 +1,94 @@
 import battlecode as bc
+from game.game_state import GC
+from pathfinding.node import Node
 import random
 
+class PathFinder:
 
-class SimplePathFinder:
-
+    def __init__(self):
+        PathFinder.directions = [bc.Direction.North, bc.Direction.Northeast, bc.Direction.East, bc.Direction.Southeast,
```

Comment:
tu sa da dat nieco ako battlecode.directions a vrati ti to rovno list vsetkych directionov

Reply:
aha, tak to je dokonca vlasnost pythonu, ze ked precastujes typ enum na list tak ti to vrati list vsetkych moznosti, cool :D 
list(bc.Direction)

to uz prerobim :)